Exercise 4/check-prime.js

```diff
@@ -0,0 +1,12 @@
+function checkPrime(number) {
+  let isPrime = false;
+  for (let initial = 2; initial < number; initial++) {
+    if (number % initial == 0) {
+      isPrime = false;
+      return isPrime;
+    } else {
+      isPrime = true;
+    }
+  }
+  return isPrime;
+}
```

Comment:
Well done! You could have used early returns to make the execution time shorter. Your code works well, but